README.md

```diff
@@ -3,11 +3,61 @@
 > :warning: **Under Development**
 
 [![Go Report Card](https://goreportcard.com/badge/github.com/openshift/managed-upgrade-operator)](https://goreportcard.com/report/github.com/openshift/managed-upgrade-operator)
-![Go Coverage](./coverage_badge.png)
+[![codecov](https://codecov.io/gh/openshift/managed-upgrade-operator/branch/master/graph/badge.svg)](https://codecov.io/gh/openshift/managed-upgrade-operator)
 [![GoDoc](https://godoc.org/github.com/openshift/managed-upgrade-operator?status.svg)](https://pkg.go.dev/mod/github.com/openshift/managed-upgrade-operator)
 [![License](https://img.shields.io/:license-apache-blue.svg)](http://www.apache.org/licenses/LICENSE-2.0.html)
 
+----
+
+The _Managed Upgrade Operator_ has been created for the [OpenShift Dedicated Platform](https://docs.openshift.com/dedicated/4/) (OSD) to manage the orchestration of automated in-place cluster upgrades.
+
+The operator's goal is to satisfy the operating conditions that a managed cluster must hold pre/post-invocation of a cluster upgrade. This may include, but is not limited to, actions such as:
+* Pre and post-upgrade health checks
+* Worker capacity scaling during the upgrade period
+* Maintenance window scheduling and notification
```

Comment:
When thinking of this operator's goals:
1. Having pre/post health checks for upgrades in addition to what CVO provides is fine. But I think we can add some high level functional points here in introduction than implementation points.
2. Worker capacity scaling seemed temporary measure so do we include it as a core operator goal? 
3. Maintenance window scheduling point we could postpone for now and add when finalized that upgrade operator is going to handle it?

Reply:
This list wasn't really meant to be a prescriptive list of the operator's goals - it was more to provide context around why the operator is needed at all and where it fits in to the process by giving examples of things that it could potentially perform. I'll reorganize and reword to hopefully make it clearer around this, thanks.